internal/server/statvar_search.go

```diff
@@ -56,31 +57,46 @@ func searchTokens(tokens []string, index *SearchIndex) ([]string, []string) {
 	}
 
 	// Only select sv and svg that matches all the tokens
-	svList := []string{}
+	svList := []*pb.EntityInfo{}
 	for sv, c := range svCount {
 		if c == len(tokens) {
-			svList = append(svList, sv)
+			svList = append(svList, &pb.EntityInfo{
+				Dcid: sv,
+				Name: index.ranking[sv].RankingName,
+			})
 		}
 	}
 	// Sort stat vars by number of PV; If two stat vars have same number of PV,
-	// then order alphabetically.
-	sort.Strings(svList)
+	// then order by the stat var (group) name.
 	sort.SliceStable(svList, func(i, j int) bool {
```

Comment:
nit: if stable order is not needed, we can just use Strings

Reply:
yea, this could help for testing